config.sample.yaml

```diff
@@ -63,6 +63,9 @@ ircService:
       ssl: true
       # Whether or not IRC server is using a self-signed cert or not providing CA Chain
       sslselfsign: false
+      # Should the connection attempt to identify via SASL (if a server or user password is given?)
```

Comment:
question should be answered

Reply:
Not sure why there is a ?. It reads fine without.